Assets/MixedRealityToolkit-Tests/MixedRealityToolkitTests.cs

```diff
@@ -178,7 +176,6 @@ public void Test11_TestMixedRealityExtensionService2DoesNotReturn()
 
             // Validate non-existent component
             MixedRealityToolkit.Instance.GetService(typeof(ITestComponent2), "Test2");
-            LogAssert.Expect(LogType.Error, "Unable to find Test2 Manager.");
```

Comment:
The test should probably not be checking the log string then (requires an exact match), and checking the return from GetService.

Reply:
That's how Unity's test suite works. 

Please fix the test.

Reply:
I will implement a less fragile version of the test by checking that the value returned by GetService is null, using Assert.IsTrue()

Reply:
> We also test the errors we "Expect" when something is not correct. 

While I agree that this is a good endeavor, it can make the tests fragile. If a method changes the log string (ex: a spelling error is fixed, the strings get localized, or if we add a feature like this one, that allows developers to disable some log messages) the test will fail.

One possible solution is to return error _codes_ and validate those values, rather than a specific log string.